test/test-rotateTetromino.js

```diff
@@ -0,0 +1,375 @@
+var assert = require('chai').assert;
+var tetris = require('../tetris');
+describe("O-tetromino", function () {
+    it("O-tetromino rotation phase", function () {
```

Comment:
All test names should reflect both initial and target phase, not only initial phase